### PR TITLE
Before you begin 1077

### LINKED ISSUE #1077: Before you begin
### Overview
We have updated the Before you begin page per Bonnie's requests. Reference #990 for more information.

### Action Items

- [x] Update Take your time title and paragraph text (mobile/desktop)
- [x] Added Keep your browser open alert (mobile/desktop)
- [x] Update spacing between alert and buttons in mobile

### Resources/Instructions
Reference issue: [1077 under Dev Team in Figma](https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?type=design&node-id=13877%3A4545&mode=design&t=k7oUNGQnNzUbM09b-1)
`

### PR DESCRIPTION
**Pull request by davidwiese**
Fixes #1077 

### Changes made
  - Updated copy on page to match figma
  - Added `Keep your browser open` alert section
  - Updated spacing between alert and buttons on mobile

### Screenshots of proposed changes

<details>
<summary>Desktop</summary>

Screen Shot 2024-01-17 at 12 45 04 PM

</details>

<details>
<summary>Mobile</summary>

Screen Shot 2024-01-17 at 12 44 32 PM

</details>

### CODE REVIEW

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
Hey @davidwiese ! This looks great! My only request is that we import and use a material UI icon instead of importing one.

[Check out this one.](https://mui.com/material-ui/material-icons/?query=alert&theme=Rounded&selected=ErrorRounded)
Check out the `Advice.tsx` for some insight on how to style and use the icons.

**Comment by davidwiese** [comment]
> Hey @davidwiese ! This looks great! My only request is that we import and use a material UI icon instead of importing one.
> 
> [Check out this one.](https://mui.com/material-ui/material-icons/?query=alert&theme=Rounded&selected=ErrorRounded) Check out the `Advice.tsx` for some insight on how to style and use the icons.

All finished for re-review

File: products/statement-generator/src/assets/Alert.png
Comment (sydneywalcoff) [inline products/statement-generator/src/assets/Alert.png:1]: Remove this and we're good! :)

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
Just remove the alert image and we're good!

File: products/statement-generator/src/assets/Alert.png
Comment (davidwiese) [reply products/statement-generator/src/assets/Alert.png:1]: @sydneywalcoff Woops. All set!

File: products/statement-generator/src/styles/utilityStyles.ts [hunk 1]
```diff
@@ -47,7 +47,15 @@ const useUtilityStyles = makeStyles<Theme>(
...
         width: '100%',
+        [breakpoints.down('sm')]: {
+          marginTop: 40,
+        },
+
+        [breakpoints.down('xs')]: {
+          marginTop: 40,
+        },
       },
+
```
Comment (sydneywalcoff) [inline products/statement-generator/src/styles/utilityStyles.ts:58]: @davidwiese jk one last thing. Is this supposed to be here? I'm not seeing a button container on the Before you Being page, and we don't want unintended consequences on the rest of the buttons?

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
For realsies, last thing lol @davidwiese

File: products/statement-generator/src/styles/utilityStyles.ts (hunk 1 above)
Comment (davidwiese) [reply products/statement-generator/src/styles/utilityStyles.ts:58]: Yea, that's adjusting the margin between the Before You Begin section and the Buttons on the bottom of the page for mobile layouts (part of the `<FlowNavigation />` component).

Comment (davidwiese) [reply products/statement-generator/src/styles/utilityStyles.ts]: @sydneywalcoff Ok I was just overthinking the problem, or was just misunderstanding the goal of the design change. 

My initial understanding was that the buttons should be a fixed distance from the alert, when really it's just the minimum distance that is desired.

Should be good to go now!

File: products/statement-generator/src/styles/utilityStyles.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ const useUtilityStyles = makeStyles<Theme>(
         flexDirection: 'row',
         width: '100%',
       },
+
```
Comment (sydneywalcoff) [inline products/statement-generator/src/styles/utilityStyles.ts:51]: @davidwiese seems silly, but lets remove this added line so you aren't making changes to this file at all.

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
@davidwiese just a tiny thing and then I'll merge

**Review by sydneywalcoff — APPROVED** [review]
@davidwiese looks good!! I'll merge now

remember to re-add the `ready for dev lead` label when its ready again after corrections.

**Pull request merged**